src/index.html

```diff
@@ -1,13 +1,392 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu header__menu-opener"></a>
+            <a href="#" class="logo">
+              <img
+              class="logo__image"
+              src="images/logo.png"
+              alt="Miami Condo Kings"
+              >
+            </a>
+            <a href="tel: +3 80676712113" class="icon icon--phone header__call-icon"></a>
+          </div>
+          <h1 class="header__title">
+            Miami <br>
+            Condo Kings
+          </h1>
+        </div>
+
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container page__body--with-menu">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="icon icon--close"></a>
+
+            <a href="#" class="logo">
+              <img
+              class="logo__image"
+              src="images/logo.png"
+              alt="Miami Condo Kings"
+              >
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact Us</a>
+            </li>
+          </ul>
+
+          <a
+          href="tel: +1 234 555-55-55"
```

Comment:
and here is another one) I think it must be the same phone number all over the page